### PR TITLE
Create SecureHeadersMiddleware.py

### LINKED ISSUE #502: Add secure headers middleware
Make a middleware available that you are able to have secure headers available to your project by simply adding it to the middleware stack:

```
from masonite.middleware import SecureHeadersMiddleware

HTTP_MIDDLEWARE = [
    ...
    SecureHeadersMiddleware,
]
```

### PR DESCRIPTION
**Pull request by vaibhavmule**
Closes #502

### CODE REVIEW

File: masonite/middleware/SecureHeadersMiddleware.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            'Strict-Transport-Security': 'max-age=63072000; includeSubdomains',
+            'X-Frame-Options': 'SAMEORIGIN',
+            'X-XSS-Protection': '1; mode=block',
+            'X-Content-Type-Options': 'nosniff',
+            'Referrer-Policy': 'no-referrer, strict-origin-when-cross-origin',
+            'Cache-control': 'no-cache, no-store, must-revalidate',
+            'Pragma': 'no-cache',
+        }
+        headers.update(middleware.SECURE_HEADERS)
+        if not DEBUG:
```
Comment (clsource) [inline masonite/middleware/SecureHeadersMiddleware.py]: Why this headers would be added only when not in DEBUG mode?.
Just need more clarification about this 👍

**Comment by josephmancuso** [comment]
Also need to add tests. Since you made a branch from this repo I can add them tonight if I get around to it. Test really would be just passing in a request object and then asserting some of the headers were set correctly.

File: masonite/middleware/SecureHeadersMiddleware.py (hunk 1 above)
Comment (vaibhavmule) [reply masonite/middleware/SecureHeadersMiddleware.py]: Actually, I have mixed feeling about it. 

I only needed it in a production environment, but I think we can remove `not Debug` and give the ability to developer comment out SecureHeadersMiddleware in `config/middleware.py`

**Comment by vaibhavmule** [comment]
@josephmancuso I wasn't  sure about branch that's why I choose 2.1

**Pull request merged**